### PR TITLE
Create flask backend

### PR DESCRIPTION
**Pull request by sohumgala**
This PR sets up the Flask backend, connects it to the MySQL database, and provides a README for how to get the backend running locally.

### CODE REVIEW

**Comment by josealtron** [comment]
The README instructions are missing instructions on setting up the dependencies (install flask, pip, etc.). Other team members may not have this setup/know how to set this up and having it in the README would save time.

File: backend/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Backend Setup
+
+First, set up database authentication, then you'll be able to run the Flask backend, which is connected to the Harvest4Us AWS RDS MySQL Database
+
+### Set up Database Authentication
+1. Create `db_auth.json` in this directory as specified by db-info.docx in team files (be sure not to commit this file, as it contains the database password).
+
```
Comment (josealtron) [inline backend/README.md:7]: Add additional installation instructions here

**Review by josealtron — CHANGES_REQUESTED** [review]
Ready to merge once installation instructions are added to README

File: backend/README.md (hunk 1 above)
Comment (sohumgala) [reply backend/README.md:15]: Done

**Pull request merged**